docs/organizer-guide/newsletter/newsletter-process.rst

```diff
@@ -120,15 +120,15 @@ Once all the story drafts have come in, it's time to assemble, based on the outl
    *Interested in sponsoring the newsletter? Take a look at our* `sponsorship prospectus </sponsorship/newsletter/>`__.
 
    ------------------
-   Featured job posts
+   Featured Job Posts
    ------------------
 
    - `TITLE <https://jobs.writethedocs.org/LINK>`__, COMPANY (LOCATION)
 
    *To apply for these jobs and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
 
    ----------------
-   Events coming up
+   Events Coming Up
    ----------------
```

Comment:
Personal preference for the lower case here, but :man_shrugging:

Reply:
Yeah, I get that. Mostly I just noticed a mixture creeping in and wanted to get away from that and picking either standard seemed better. It's also something of a 🇺🇸 vs. 🇬🇧 thing. 🤷

Reply:
For sure your call, and for sure consistency is better than a mix.